src/main/java/core/basesyntax/IllegalArgumentException.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+class IllegalArgumentException extends RuntimeException {
```

Comment:
we don't need to create our own exception, use existing one 
so you can just delete this class